.github/workflows/release-please.yml

```diff
@@ -18,6 +18,32 @@ jobs:
           signoff: "OpenFeature Bot <109696520+openfeaturebot@users.noreply.github.com>"
     outputs:
       release_created: ${{ steps.release.outputs.releases_created }}
+      all: ${{ toJSON(steps.release.outputs) }}
+      paths_released: ${{ steps.release.outputs.paths_released }}
+
+  sbom:
+    needs: release-please
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        release: ${{ fromJSON(needs.release-please.outputs.paths_released) }}
```

Comment:
Will this be empty if a release wasn't created? If so, how will this behave?

Reply:
Yes and you are right, this would have let to an error.
I added a condition that checks this now.